src/deploy-cloudrun.ts

```diff
@@ -193,25 +217,24 @@ export async function run(): Promise<void> {
       throw new Error(`failed to execute gcloud command \`${commandString}\`: ${errMsg}`);
     }
 
-    // gcloud outputs status information to stderr.
-    // TODO: update this to use JSON or YAML machine-readable output instead.
-    setUrlOutput(output.stdout + output.stderr);
+    // Map outputs by response type
+    const outputs: DeployCloudRunOutputs =
+      responseType === ResponseTypes.UPDATE_TRAFFIC
+        ? parseUpdateTrafficResponse(output.stdout)
+        : parseDeployResponse(output.stdout, { tag });
```

Comment:
```suggestion
        : parseDeployResponse(output.stdout, { tag: tag });
```

I dislike this particular node magic.